frontend/src/hooks/useLayerQueryProvider.ts

```diff
@@ -0,0 +1,68 @@
+import { AxiosResponse } from 'axios';
+import { useCallback } from 'react';
+import { toast } from 'react-toastify';
+
+import { useApiGeocoder } from './pims-api/useApiGeocoder';
+import { useApiRequestWrapper } from './pims-api/useApiRequestWrapper';
+import { IGeocoderPidsResponse, IGeocoderResponse } from './useApi';
+
+export const useGeocoderRepository = () => {
+  const { getSitePidsApi, searchAddressApi, getNearestToPointApi } = useApiGeocoder();
```

Comment:
This file looks like a duplicate of `useGeocoderRepository.ts` ...

Reply:
You are right. I actually used that on the container